src/compiler/path.rs

```diff
@@ -0,0 +1,129 @@
+use crate::{last_or_undefined, CharacterCodes, Path};
```

Comment:
This file could mostly be replaced with https://doc.rust-lang.org/stable/std/path/index.html (which is presumably pretty robustly tested and cross-platform).

Reply:
Ya this one could be interesting depending on especially how differences between `String`/`OsString`/`Path` are relevant for real use cases